extensions/amp-minute-media-player/0.1/amp-minute-media-player.js

```diff
@@ -0,0 +1,467 @@
+    this.preconnect.url('https://www.oo-syringe.com', onLayout);
+  }
+
+  /** @override */
+  buildCallback() {
+    this.initFields_();
+
+    const deferred = new Deferred();
+    this.playerReadyPromise_ = deferred.promise;
+    this.playerReadyResolver_ = deferred.resolve;
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return (
+      layout == Layout.FILL ||
+      layout == Layout.FIXED ||
+      layout == Layout.FIXED_HEIGHT ||
+      layout == Layout.FLEX_ITEM ||
+      layout == Layout.NODISPLAY ||
+      layout == Layout.RESPONSIVE
+    );
+  }
+
+  /**
+   * Init params
+   * @private
+   */
+  initFields_() {
+    const {element} = this;
+    this.contentType_ = userAssert(
+      element.getAttribute('data-content-type'),
+      'The data-content-type must be specified for <amp-minute-media-player> %s',
+      element
+    );
+
+    if (this.contentType_ != 'semantic') {
+      this.contentId_ = userAssert(
+        element.getAttribute('data-content-id'),
+        'The data-content-id must be specified for %s ' +
+          'data-content-type in <amp-minute-media-player> %s',
+        this.contentType_,
+        element
+      );
+    } else {
+      this.initSemanticFields_();
+    }
+  }
+
+  /**
+   * Init Semantic params
+   * @private
+   */
+  initSemanticFields_() {
+    const {element} = this;
+    this.scannedElement_ = element.getAttribute('data-scanned-element') || '';
+    this.tags_ = element.getAttribute('data-tags') || '';
+    this.minimumDateFactor_ =
+      element.getAttribute('data-minimum-date-factor') || '';
+    this.scannedElementType_ =
+      element.getAttribute('data-scanned-element-type') || '';
+  }
+
+  /**
+   * @param {!Event} event
+   * @private
+   */
+  handleMinuteMediaPlayerMessage_(event) {
+    if (!originMatches(event, this.iframe_, 'https://www.oo-syringe.com')) {
+      return;
+    }
+    const eventData = getData(event);
+    if (!isJsonOrObj(eventData)) {
+      return;
+    }
+    const data = objOrParseJson(eventData);
+    if (data === undefined) {
+      return; // We only process valid JSON.
+    }
+
+    redispatch(this.element, data['event'], {
+      'ready': VideoEvents.LOAD,
+      'playing': VideoEvents.PLAYING,
+      'pause': VideoEvents.PAUSE,
+      'ended': [VideoEvents.ENDED, VideoEvents.PAUSE],
+      'ads-ad-started': VideoEvents.AD_START,
+      'ads-ad-ended': VideoEvents.AD_END,
+    });
+
+    if (data['event'] === 'mute') {
+      const muted = data['value'];
+      if (this.muted_ == muted) {
+        return;
+      }
+      this.muted_ = muted;
+      this.element.dispatchCustomEvent(mutedOrUnmutedEvent(this.muted_));
+      return;
+    }
+  }
+
+  /**
+   * Build Iframe source
+   * @return {string}
+   * @private
+   */
+  iframeSource_() {
+    const baseUrl =
+      'https://www.oo-syringe.com/dev/amp/mplayer.html' +
+      `?content_type=${encodeURIComponent(this.contentType_)}`;
```

Comment:
set `content_type` as part of the query params dict.

Reply:
Done